tests/datasets/test_mapinwild.py

```diff
@@ -0,0 +1,128 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import glob
+import os
+import shutil
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+import pytest
+import torch
+import torch.nn as nn
+from _pytest.monkeypatch import MonkeyPatch
+from torch.utils.data import ConcatDataset
+
+import torchgeo.datasets.utils
+from torchgeo.datasets import MapInWild
+
+
+def download_url(url: str, root: str, *args: str, **kwargs: str) -> None:
+    shutil.copy(url, root)
+
+
+class TestMapInWild:
+    @pytest.fixture(params=["train", "validation", "test"])
```

Comment:
You aren't actually using this parameterization anywhere are you? I think you need to add a request parameter to the function and use it to define the split. At the moment, you're just testing the "train" split 3 times lol.

Reply:
Done :)

Reply:
Can you push this change?